modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/application/ApplicationDescriptorFactoryTestCase.java

```diff
@@ -91,6 +91,11 @@ private static File getResourceFile(String resource) {
                                                                         .getAbsolutePath());
 
   @Rule
+  public SystemProperty remoteMavenRepository =
+      new SystemProperty("muleRuntimeConfig.maven.repositories.mulesoftPublic.url",
```

Comment:
The only thing about this test that make it seem like an unit test is that extends AbstractMuleTestCase. But is actually testing the whole creation of an application descriptor from an application folder structure.